### PR TITLE
Update color

### PR DESCRIPTION
**Pull request by HsupoLeng**

### CODE REVIEW

File: src/DemoPackageJH.jl [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module DemoPackageJH
 
-print_greeting() = print("Hello, world!")
+print_greeting() = printstyled("Hello, world!"; color=:magenta)
```
Comment (Junxiao-Hou) [inline src/DemoPackageJH.jl]: Would this look better in `::blue`?

**Review by Junxiao-Hou — COMMENTED** [review]
Thanks for the pull request! This offers great improvement. I just have one small suggestion (see below).

File: src/DemoPackageJH.jl (hunk 1 above)
Comment (HsupoLeng) [reply src/DemoPackageJH.jl]: Sure enough blue it will be.

**Comment by Junxiao-Hou** [comment]
Thanks for your contribution!

**Pull request merged**